src/dodal/beamlines/adsim.py

```diff
@@ -0,0 +1,38 @@
+from dodal.common.beamlines.beamline_utils import (
+    device_instantiation,
+    get_directory_provider,
+)
+from dodal.common.beamlines.beamline_utils import set_beamline as set_utils_beamline
+from dodal.devices.adsim import SimStage
+from dodal.devices.areadetector import AdSimDetector
+from dodal.log import set_beamline as set_log_beamline
+from dodal.utils import get_beamline_name, get_hostname
+
+BL = get_beamline_name(get_hostname())
+set_log_beamline(BL)
+set_utils_beamline(BL)
+
+
+def stage(
+    wait_for_connection: bool = True, fake_with_ophyd_sim: bool = False
+) -> SimStage:
+    return device_instantiation(
+        SimStage,
+        "sim_motors",
```

Comment:
I'd have device.name and the function share the same name, so either make device.name "stage" or the function renamed to "sim_motors"